.gitignore

```diff
@@ -130,3 +130,4 @@ zz_generated_*_test.go
 
 # generated by verify-godeps.sh
 vendordiff.patch
+/.metadata/
```

Comment:
we shouldn't touch the .gitignore file as part of this PR.

Reply:
I have reverted it.